docs/data/base/components/textarea-autosize/textarea-autosize.md

```diff
@@ -39,9 +39,9 @@ export default function MyApp() {
 
 Textarea Autosize behaves similarly to the native HTML`<textarea>`.
 
-By default, an empty Textarea Autosize component renders as a single row, as shown in the following demo:
+By default, the Textarea Autosize component renders as a single row when it is empty. In the following demo, you can observe this behavior and learn how to apply custom styles to the component.
```

Comment:
```suggestion
By default, an empty Textarea Autosize component renders as a single row, as shown in the following demo:
```

I think we can keep the previous phrasing as in the future all demos will have the Tailwind CSS + Plain CSS switch.